ui/raidboss/data/06-ew/raid/p6s.ts

```diff
@@ -48,6 +80,51 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: NetRegexes.startsUsing({ id: '7881', source: 'Hegemone', capture: false }),
       response: Responses.goSides(),
     },
+    {
+      id: 'P6S Exchange of Agonies Markers',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({}),
+      condition: (data, matches) => {
+        return data.me === matches.target && !(/(00(?:4F|5[0-6]))|013E|0148|0065/).test(getHeadmarkerId(data, matches));
+      },
+      infoText: (data, matches, output) => {
+        const correctedMatch = getHeadmarkerId(data, matches);
+        switch (correctedMatch) {
+          case '0163':
+          case '0167':
```

Comment:
Somebody else kindly sent me:

163    Stack
167    Stack
169    Stack

164    Spread
165    Spread
16A    Spread

166    Dynamo
168    Dynamo
16E    Dynamo

Are these missing? It's possible some of them are from the second application, in which case maybe the callout should just be "stack" and not "stack donut"?

Reply:
Ah, it's possible I was missing some headmarkers as I only checked three logs.